src/components/admin/Billing/index.tsx

```diff
@@ -79,17 +77,8 @@ function AdminBilling({ showAddPayment }: AdminBillingProps) {
         return { start: startMonth, end: currentMonth };
     }, [currentMonth]);
 
-    const {
-        billingStats,
-        error: billingStatsError,
-        isValidating: isValidatingStats,
-    } = useBillingCatalogStats();
-
-    const { invoices: currentMonthInvoices, isValidating: isValidatingRecord } =
-        useInvoice(startOfMonth(currentMonth));
```

Comment:
We do not need to constantly ping the server and get the latest data so removing.